index.html

```diff
@@ -1,51 +1,236 @@
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="data">
+            <div class="content">
+              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
+              <div class="language">
+                <p>JavaScript</p>
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Ruby on rails</p>
+              </div>
+              <div class="wrapper">
+                <button type="button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- About Myself -->
+  <section class="about-me">
+    <div class="title">
+      <div class="img-content">
+        <h1>About me</h1>
+        <img src="./images/mask.svg" alt="" />
+      </div>
+      <p>
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="button-content">
+        <img src="./images/second-mask.svg" alt="button" />
+        <button>Get My Resume</button>
+      </div>
+    </div>
+  </section>
+  <!-- Libraries -->
+  <section class="lib-container">
+    <div class="lib-grid">
+      <div class="lib-card">
+        <img src="./images/icons/Icons1.png" alt="" />
+        <h1>Languages</h1>
+        <ul>
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
         </ul>
-      </section>
+      </div>
+      <div class="lib-card">
+        <img src="./images/icons/Icons2.png" alt="" />
+        <h1>Frameworks</h1>
+        <ul>
+          <li>Boostrap</li>
+          <li>Ruby on rail</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
```

Comment:
- [ ] I will suggest you add `RSpec` to the list. So as to match the design.

Figma design:
![image](https://user-images.githubusercontent.com/51296741/143619195-22b73685-aa92-4fb9-9dc2-99af8cea9908.png)

Your design:
![image](https://user-images.githubusercontent.com/51296741/143619246-01adb07d-76c3-4f4e-93cd-90184c5c475c.png)

Reply:
ok fixed. Thank you.